tests_v2/unit/app/utils/paginate/test_legacy_behavior.py

```diff
@@ -0,0 +1,46 @@
+from unittest.mock import AsyncMock, Mock
+
+import pytest
+
+from app.models.orm.base import Base as ORMBase
+from app.utils.paginate import paginate_collection
+
+
+@pytest.mark.asyncio
+async def test_legacy_no_pagination_happens_for_default_values():
```

Comment:
Now I see what you mean by making lots of tiny tests that only test one thing! Def will try to do this more going forward